src/js/index.js

```diff
@@ -7,36 +7,37 @@ function getDiscussTopics() {
 	if (!section || !discussions) {
 		return;
 	}
+	let idList = [];
 	fetch(`${Discussion.ORIGIN}/latest.json`).then(r => r.json()).then(r => {
 		const topics = r.topic_list.topics.slice(0, 2);
-		const allUsers = new Map(r.users.map(user => [user.id, user]));
-
-		if (topics && topics.length && allUsers.size) {
-			drawTopics(topics, allUsers);
-		}
-	}).then(_ => {
-		return fetch(`${Discussion.ORIGIN}/top.json`);
-	}).then(r => r.json()).then(r => {
-		const topics = r.topic_list.topics.slice(0, 3);
-		const allUsers = new Map(r.users.map(user => [user.id, user]));
-
-		if (topics && topics.length && allUsers.size) {
-			drawTopics(topics, allUsers);
-		}
-	}).then(_ => {
-		section.classList.remove('hidden');
-	});
+		topics.map((topic) => idList.push(topic.id));
```

Comment:
`map` is best for mutating the array. `forEach` is more appropriate here.

```suggestion
		topics.forEach(topic => latestTopicIds.add(topic.id));
```